source/users/sessions.txt

```diff
@@ -0,0 +1,473 @@
+.. _manage-user-sessions:
+
+====================
+Manage User Sessions
+====================
+
+.. contents:: On this page
+   :depth: 2
+
+Overview
+--------
+
+App Services manages :ref:`user sessions <user-sessions>` with access
+tokens and refresh tokens. An access token allows anyone that has the
+token to use the session it's associated with for up to
+{+user-session-expiration-time+}. A refresh token lets you generate a
+new access token for a session even if the previous token has already
+expired.
+
```

Comment:
i think this page could use a brief discussion of why you might want to get a user access token, in other words: what can you access with it? 

off top of my head, this would be:

- data api
- custom https endpoints
- graphql api

not sure if missing anything here.